src/components/Input/Input.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "styled-components/macro";
+import Input from "./Input";
+
+export default {
+  component: Input,
+  title: "Input",
+};
+
+const StorybookWhiteCentered = {
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+  width: "100%",
+  height: "100%",
+  background: "white",
+};
+
+export const InputStory = () => (
+  <div style={StorybookWhiteCentered}>
+    <Input type="text" placeholder="Name"></Input>
```

Comment:
Optional suggestion, but you can shorten this to
```suggestion
    <Input type="text" placeholder="Name" />
```